### DISCUSSION TITLE
error in generating protein ligand interaction fingerprint

### DISCUSSION THREAD

**Discussion by jjmallen** [MDAnalysis/mdanalysis#4611] [Q&A]
Good day

I am trying to generate protein ligand interaction fingerprint for my system and i am getting this error. I have tried so many options and I didn't succeed.kindly assist 

```python
import MDAnalysis as mda
import MDAnalysis.topology.guessers as guessers
import prolif as plf
import prolif.plotting as plf_plot

#load topology and trajectory
u = mda.Universe('1ns.gro', '1ns.xtc')

#create selections for ligand and protein
ligand_selection = u.select_atoms("resname LIG")
protein_selection = u.select_atoms("protein")
ligand_selection.guess_bonds()
protein_selection.guess_bonds()

ligand_selection, protein_selection

# Check if the selection contains atoms
if protein_selection.n_atoms == 0:
    raise ValueError("Protein selection is empty, please check your selection string.")

# Guess the elements from other attributes
guessers.guess_types(protein_selection)
```

<img width="1070" alt="Screenshot 2024-06-04 at 15 44 16" src="https://github.com/MDAnalysis/mdanalysis/assets/127143302/ad6cb32c-9ad2-473a-a24a-f1b9182b7766">
<img width="1070" alt="Screenshot 2024-06-04 at 15 44 16" src="https://github.com/MDAnalysis/mdanalysis/assets/127143302/756fcbe9-cd2d-4611-a0cb-7263c4d6bd20">

**Comment by RMeli** [comment]
Hi @jjmallen. Unfortunately it is not easy to help you with the screenshot you posted, since the error is incomplete. We can just see it's an `AttributeError` and where it's happening. Can you please show us the whole error message?

Please copy-paste the error in a [code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks), which is searchable. Unfortunately, images are not (yet) searchable on GitHub.
